### ISSUE TITLE
FR: Add `autosaveInterval` to the plugin settings

### ISSUE THREAD

**Issue by dyzdyz010** [zsviczian/obsidian-excalidraw-plugin#888]
**Is your feature request related to a problem? Please describe.**
The `autosaveInterval` property is in different default values with mobile & desktop environments, causing meaningless commits in git. Currently not possible to manually specify auto save interval in plugin settings.

**Describe the solution you'd like**
Add a setting field for `autosaveInterval` in the plugin settings

**Describe alternatives you've considered**
Use the same `autosaveInterval` value for different platforms

**Additional context**

**Comment by zsviczian** [comment]
In 1.8.1 I have added plugin settings to allow for less frequent autosave intervals. There is a separate setting for desktop and a separate one for mobile. Will release it in the coming days.

**Comment by chrisklpp** [comment]
I just installed Excalidraw (V 2.1.3) and still have the same issue.
My obsidian-excalidraw-plugin/data.json still has three lines:
```
  "autosaveInterval": 15000,
  "autosaveIntervalDesktop": 15000,
  "autosaveIntervalMobile": 10000,
```
The first line is automatically updated depending on which device I used (laptop or iPad), creating the unnecessary repeating commits.

**Comment by jmhammond** [comment]
I wanted to follow up to see if this would be fixed because as @chrisklpp points out, each time I open a different device -- laptop and ipad -- the `data.json` file updates the `autosaveInterval` to whichever device it is on. This results in a (pointless) commit to the file, since it'll just change again on the other device.

**Comment by zsviczian** [comment]
I understand. This is a legacy issue because I originally only had the first setting... then, I kept it for backward compatibility. I'll clean this up. Please open a new issue for this, as this is a closed issue from 1.5 years ago, this will certainly escape my attention. Unfortunately, the fact is, there are over 650 open items in the issue list, and only one of me... so inevitably items at the bottom, especially items already closed never get addressed.
